gnat2goto/driver/tree_walk.adb

```diff
@@ -861,48 +861,36 @@ package body Tree_Walk is
          end;
       end Make_Case_Test;
 
-      Case_Body_Leaf : Irep := Ireps.Empty;
-      Case_Body_Root : Irep := Ireps.Empty;
-      This_Alt : Node_Id := First (Alternatives (N));
+      function Make_Case_If_Expression (Alternatives : Node_Id) return Irep;
+      function Make_Case_If_Expression (Alternatives : Node_Id) return Irep
+      is
+         This_Alt : constant Node_Id := Alternatives;
```

Comment:
Do you still need this node?

Reply:
So, technically not functionality wise, but I wanted to rename it to distinguish between the list of alternatives and the thing that's currently pointing to the head of that list (even though they're the same thing as far as the implementation is concerned)